.circleci/dependencies.txt

```diff
@@ -0,0 +1,231 @@
+perl-DBIx-Class-DynamicDefault-0.04
+perl-DBIx-Class-OptimisticLocking-0.02
+perl-DBIx-Class-Schema-Config-0.001011
+perl-Devel-Cover-1.29
+perl-Devel-Cover-Report-Codecov-0.25
+perl-Devel-GlobalDestruction-0.14
+perl-Devel-OverloadInfo-0.005
+perl-Devel-StackTrace-2.03
+perl-Digest-HMAC-1.03
+perl-Digest-SHA1-2.13
+perl-Dist-CheckConflicts-0.11
+perl-Email-Address-1.912
+perl-Encode-Locale-1.05
+perl-Eval-Closure-0.14
+perl-Exception-Class-1.44
+perl-Export-Attrs-0.1.0
+perl-Exporter-Declare-0.114
+perl-Exporter-Tiny-1.000000
+perl-File-Copy-Recursive-0.38
+perl-File-HomeDir-1.002
+perl-File-Listing-6.04
+perl-File-Path-2.150000
+perl-File-pushd-1.014
+perl-File-ShareDir-1.104
+perl-File-Touch-0.11
+perl-File-Which-1.22
+perl-Furl-3.13
+perl-Hash-Merge-0.200
+perl-HTML-Parser-3.72
+perl-HTML-Tagset-3.20
+perl-HTTP-Cookies-6.04
+perl-HTTP-Daemon-6.01
+perl-HTTP-Date-6.02
+perl-HTTP-Message-6.14
+perl-HTTP-Negotiate-6.01
+perl-HTTP-Parser-XS-0.17
+perl-Import-Into-1.002005
+perl-indirect-0.38
+perl-IO-HTML-1.001
+perl-IO-Socket-SSL-2.052
+perl-IO-String-1.08
+perl-IO-stringy-2.111
+perl-IO-Tty-1.12
+perl-IPC-Run-0.99
+perl-IPC-Run3-0.048
+perl-IPC-System-Simple-1.25
+perl-JavaScript-Minifier-XS-0.11
+perl-JSON-2.97001
+perl-JSON-Validator-3.08
+perl-JSON-XS-3.04
+perl-libwww-perl-6.31
+perl-Lingua-EN-Inflect-1.903
+perl-List-MoreUtils-0.428
+perl-List-MoreUtils-XS-0.428
+perl-local-lib-2.000023
+perl-Log-Contextual-0.008001
+perl-LWP-MediaTypes-6.02
+perl-Meta-Builder-0.003
+perl-Minion-9.13
+perl-Minion-Backend-SQLite-4.005
+perl-Module-Build-0.422400
+perl-Module-Find-0.13
+perl-Module-Implementation-0.09
+perl-Module-Pluggable-5.2
+perl-Module-Runtime-0.016
+perl-Module-Runtime-Conflicts-0.003
+perl-Mojo-IOLoop-ReadWriteProcess-0.23
+perl-Mojolicious-8.23
+perl-Mojolicious-Plugin-AssetPack-2.08
+perl-Mojo-Pg-4.16
+perl-Mojo-RabbitMQ-Client-0.3.1
+perl-Mojo-SQLite-3.002
+perl-Moo-2.003004
+perl-Moose-2.2010
+perl-MooseX-Role-Parameterized-1.10
+perl-MRO-Compat-0.13
+perl-multidimensional-0.014
+perl-namespace-autoclean-0.28
+perl-namespace-clean-0.27
+perl-Net-AMQP-0.06
+perl-Net-HTTP-6.17
+perl-Net-IP-1.26
+perl-Net-OpenID-Common-1.20
+perl-Net-OpenID-Consumer-1.18
+perl-Net-SNMP-6.0.1
+perl-Net-SSH2-0.69
+perl-Net-SSLeay-1.81
+perl-Package-DeprecationManager-0.17
+perl-Package-Stash-0.37
+perl-Package-Stash-XS-0.28
+perl-Package-Variant-1.003002
+perl-PadWalker-2.3
+perl-Params-Util-1.07
+perl-Params-Validate-1.29
+perl-Params-ValidationCompiler-0.26
+perl-Parse-PMFile-0.41
+perl-Parse-RecDescent-1.967015
+perl-Path-Class-0.37
+perl-Path-Tiny-0.104
+perl-Perl-Critic-1.130
+perl-Perl-Critic-Freenode-0.026
+perl-Perl-Critic-Pulp-96
+perl-Perl-Tidy-20190601
+perl-Pod-MinimumVersion-50
+perl-Pod-POM-2.01
+perl-Pod-Spell-1.20
+perl-PPI-1.236
+perl-PPIx-Regexp-0.058
+perl-PPIx-Utilities-1.001000
+perl-Readonly-2.05
+perl-Regexp-Common-2017060201
+perl-Role-Tiny-2.000006
+perl-Scope-Guard-0.21
+perl-Selenium-Remote-Driver-1.27

```

Comment:
Please change this to `perl-Selenium-Remote-Driver-1.33` (see https://progress.opensuse.org/issues/56810#note-18). The link is already in our devel repo so this should work, right?

Reply:
The bot should submit PR for that - it will be good test for it ^^